hw03_bandyc.py

```diff
@@ -0,0 +1,106 @@
+#################################################################################
+# Author: Cody Bandy
+# Username: bandyc
+#
+# Assignment: HW03
+# Purpose: Draw a colorful house using functions.
+# Google Doc Link: https://docs.google.com/document/d/1NKZj9D6Pb3AakT7FzAY3_hwjKZ2PUx8db1sWfIzOIiU/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
```

Comment:
Cody, your code is nearly perfect! 

Because we want to exercise our Git processes, I am going to ask you to remove one of these blank spaces after the Acknowledgements, so you can see the PR review process. It is not actually a problem in your code, but typically I can find something. 

GREAT JOB!